.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: OpenWISP Radius CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        django-version:
+          - django~=2.2
+          - django~=3.0
+          - django~=3.1
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install python dependencies
+        run: |
+          pip install -U pip wheel
+          pip install ${{ matrix.django-version }}
+          pip install -U -r requirements-test.txt
+
+      - name: Install npm dependencies
+        run: sudo npm install -g jslint
+
+      - name: Install openwisp-radius
+        run: pip install -e .
```

Comment:
Swapped out `python setup.py -q develop` as it was not resolving `djangorestframework` dependency properly. Somehow, `djangorestframework==3.12.2` was getting installed in the environment.